test/simulator/test/org/apache/cassandra/simulator/test/ShortPaxosSimulationTest.java

```diff
@@ -40,3 +40,4 @@ public void selfReconcileTest() throws IOException
         PaxosSimulationRunner.main(new String[] { "reconcile", "-n", "3..6", "-t", "1000", "-c", "2", "--cluster-action-limit", "2", "-s", "30", "--with-self" });
     }
 }
+
```

Comment:
So this was the change to trigger the repeated tests, thanks. You can remove it on commit :-)